demos/wasm/src/tic80.zig

```diff
@@ -29,6 +29,13 @@ const RGBColor = extern struct {
     b: u8,
 };
 
+const TextureSource = enum(i32) {
+    TILES = 0,
+    MAP,
+    VBANK1
```

Comment:
It is not necessary to use `VBANK1`, it can also be `VBANK0`.
So `textri()` is using an inactive vbank for rendering, maybe just call it `VBANK`?

Reply:
vbank(0) would use bank 1 and vbank(1) would use bank 0?

I'll reconsider the name.  Technically it's SCREEN_INACTIVE_VBANK or some such. :-)

Reply:
ok :)